runtime/ztests/op/summarize/collect-union.yaml

```diff
@@ -0,0 +1,11 @@
+zed: summarize collect(this) | sort this
+
+vector: true
+
+input: |
+  {a:1}
+  {a:2}
+  {b:1.5}
```

Comment:
Add a `null` so we know we're doing the right thing with it?

Reply:
Hopefully ignoring it